misc/scripts/deploy-tools.sh

```diff
@@ -89,6 +94,49 @@ function deployplugin {
     done
 }
 
+function deploypluginwindows {
+    log "Compressing source into $PLUGIN_SRC_ZIP..."
+    cd $PLUGIN_SRC_DIR
+    rm -f $PLUGIN_SRC_ZIP
+    stashName=`git stash create`;
+    git archive --format zip --output $PLUGIN_SRC_ZIP $stashName
+
+    for ip in $IP_LIST
+    do
+        TARGET=root@$ip
+
+        log "Cleaning up older files from $TARGET..."
+        $SSH $TARGET <<-EOF
+            powershell Remove-Item -Recurse -Force $WIN_PLUGIN_SRC_DIR
```

Comment:
Nit: Is there a way to avoid printing this statement out on the console? Not a big concern though since this is just used internally for build process.